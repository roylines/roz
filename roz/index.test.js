const {handler} = require('.');
const roz = require('./lib/roz');

jest.mock('./lib/roz');

describe('index', () => {
  it('should call roz', async () => {
    roz.handler.mockResolvedValue('RESP');
    const response = await handler('REQ');

    expect(roz.handler.mock.calls).toMatchSnapshot();
    expect(response).toMatchSnapshot();
  });
});
