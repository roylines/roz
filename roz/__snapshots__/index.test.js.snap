// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should call roz 1`] = `
Array [
  Array [
    "REQ",
  ],
]
`;

exports[`index should call roz 2`] = `"RESP"`;
